qiita_ware/private_plugin.py

```diff
@@ -377,6 +378,45 @@ def delete_analysis(job):
         job._set_status('success')
 
 
+def list_remote_files(job):
+    """Lists valid study files on a remote server
+
+    Parameters
+    ----------
+    job : qiita_db.processing_job.ProcessingJob
+        The processing job performing the task
+    """
+    with qdb.sql_connection.TRN:
+        url = job.parameters.values['url']
+        private_key = job.parameters.values['private_key']
+        try:
+            list_remote(url, private_key)
```

Comment:
*From my previous review*:

What is happening to the output of this function, it seems like it is not being utilized anywhere and that this job does nothing(?)

Reply:
... also forgot to add a message ... this will store the information in redis so the GUI can retrieve it ... I'm working on this PR and should be ready pretty soon.

Reply:
I see, so the redis support will be added in the next PR?

Reply:
yup, yup ...